### ISSUE TITLE
Configure Base Nomad URL

### ISSUE THREAD

**Issue by JoelProminic** [Moonshine-IDE/Super.Human.Portal#44]
As we work towards using Super.Human.Portal as a customer/organization portal, we would like to support a central Nomad server that may not necessarily be on the same server.  To support this, I should add a configuration value for the base Nomad URL.

We should watch for CORS errors for this, but I don't think it matters when opening new tabs/pages.

**Comment by JoelProminic** [comment]
I added support for a configured base URL for nomad:  `nomad_base_url`

Case | Behavior
-- | --
Missing | Use old default
Blank | Use old default
Valid URL | Use provided URL
Invalid URL | Use provided URL - TODO:  do some validation?

Testing with the Prominic URLs, I'm seeing some remaining issues with escaping nested views.   I'll need to do some more investigation for this - I thought I had solved it before.
